src/map/clif.c

```diff
@@ -7564,8 +7564,8 @@ static void clif_sendegg(struct map_session_data *sd)
 	nullpo_retv(sd);
 
 	fd = sd->fd;
-	if (battle_config.pet_no_gvg && map_flag_gvg2(sd->bl.m)) { //Disable pet hatching in GvG grounds during Guild Wars [Skotlex]
-		clif->message(fd, msg_sd(sd, 866)); // "Pets are not allowed in Guild Wars."
+	if (map->list[sd->bl.m].flag.nopet) { //Disable pet hatching in GvG grounds during Guild Wars [Skotlex]
```

Comment:
* Add a space between `//` and the comment text.
 * Remove ` during Guild Wars [Skotlex]`
 * Add a punctuation character.

-> `// Disable pet hatching in GvG grounds.`

Reply:
fixed